lib/davinci_dtr_test_kit/docs/dtr_light_ehr_suite_description_v201.md

```diff
@@ -18,12 +18,21 @@ Once the connection between the DTR SMART App and the DTR Light EHR is establish
 
 ## Running the Tests
 
-If you would like to try out the tests but don't have a DTR payer server implementation, you can run these tests against the [public instance of the Inferno Reference Server](https://inferno.healthit.gov/reference-server/r4/) by using the Inferno Reference Server preset in the test suite.
+If you would like to try out the tests but don't have a DTR Light EHR implementation, you can run these tests against the [public instance of the Inferno Reference Server](https://inferno.healthit.gov/reference-server/r4/) by using the Inferno Reference Server preset in the test suite.
 
-In order to get the Inferno QA Reference Server to do an EHR launch, navigate to https://inferno.healthit.gov/reference-server/app/app-launch and use https://inferno.healthit.gov/custom/smart/launch as the App Launch URL.
+In order to get the Inferno Reference Server to do an EHR launch, navigate to https://inferno.healthit.gov/reference-server/app/app-launch and use https://inferno.healthit.gov/custom/smart/launch as the App Launch URL.
```

Comment:
I think its going to be `https://inferno.healthit.gov/suites/custom/smart/launch` (see https://github.com/onc-healthit/onc-certification-g10-test-kit/wiki/Walkthrough#step-5-perform-ehr-practitioner-app-tests)

Reply:
Addressed in [Suite description changes](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35/commits/dcec5709c38a65552ae01ee64edfa0be090dcddc)